app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt

```diff
@@ -28,15 +26,13 @@ class MultiselectSelectionMenu(
                 context.getString(R.string.tab_tray_multiselect_menu_item_bookmark),
                 textColorResource = R.color.primary_text_normal_theme
             ) {
-                context.components.analytics.metrics.track(Event.TabsTraySaveToCollectionPressed)
                 onItemTapped.invoke(Item.BookmarkTabs)
             },
 
             SimpleBrowserMenuItem(
                 context.getString(R.string.tab_tray_multiselect_menu_item_close),
                 textColorResource = R.color.primary_text_normal_theme
             ) {
-                context.components.analytics.metrics.track(Event.TabsTrayShareAllTabsPressed)
```

Comment:
Also curious why we were removing the telemetry tracking for these browser menu?

Reply:
This telemetry was just wrong -> this menu item actually closes selected tabs, but we don't have a telemetry ticket for these events yet